front/app/containers/Admin/settings/customize/index.tsx

```diff
@@ -551,14 +552,14 @@ class SettingsCustomizeTab extends PureComponent<
         <form onSubmit={this.save}>
           <Section key={'branding'}>
             <SectionTitle>
-              <FormattedMessage {...messages.titleHomepageStyle} />
```

Comment:
Don't we need to remove `titleHomepageStyle` and `subtitleHomepageStyle` from messages and locales?
It doesn't seem to be used anywhere anymore.